armsrc/pcf7931.c

```diff
@@ -141,245 +140,364 @@ int DemodPCF7931(uint8_t **outBlocks) {
+static void RealWritePCF7931(uint8_t *pass, uint16_t init_delay, int32_t l, int32_t p, uint8_t address, uint8_t byte, uint8_t data)
 {
-
 	uint32_t tab[1024]={0}; // data times frame
 	uint32_t u = 0;
 	uint8_t parity = 0;
 	bool comp = 0;
 
 	//BUILD OF THE DATA FRAME
-
 	//alimentation of the tag (time for initializing)
 	AddPatternPCF7931(init_delay, 0, 8192/2*T0_PCF, tab);
-
-	//PMC
-	Dbprintf("Initialization delay : %d us", init_delay);
 	AddPatternPCF7931(8192/2*T0_PCF + 319*T0_PCF+70, 3*T0_PCF, 29*T0_PCF, tab);
-
-	Dbprintf("Offsets : %d us on the low pulses width, %d us on the low pulses positions", l, p);
-
 	//password indication bit
 	AddBitPCF7931(1, tab, l, p);
-
-
-	//password (on 56 bits)
-	Dbprintf("Password (LSB first on each byte) : %02x %02x %02x %02x %02x %02x %02x", pass1,pass2,pass3,pass4,pass5,pass6,pass7);
-	AddBytePCF7931(pass1, tab, l, p);
-	AddBytePCF7931(pass2, tab, l, p);
-	AddBytePCF7931(pass3, tab, l, p);
-	AddBytePCF7931(pass4, tab, l, p);
-	AddBytePCF7931(pass5, tab, l, p);
-	AddBytePCF7931(pass6, tab, l, p);
-	AddBytePCF7931(pass7, tab, l, p);
-
+	// password (on 56 bits)
+	AddBytePCF7931(pass[0], tab, l, p);
+	AddBytePCF7931(pass[1], tab, l, p);
+	AddBytePCF7931(pass[2], tab, l, p);
+	AddBytePCF7931(pass[3], tab, l, p);
+	AddBytePCF7931(pass[4], tab, l, p);
+	AddBytePCF7931(pass[5], tab, l, p);
+	AddBytePCF7931(pass[6], tab, l, p);
 	//programming mode (0 or 1)
 	AddBitPCF7931(0, tab, l, p);
-
+	
 	//block adress on 6 bits
-	Dbprintf("Block address : %02x", address);
-	for (u=0; u<6; u++)
+	for (u = 0; u < 6; ++u)
 	{
-		if (address&(1<<u)) {	// bit 1
-			 parity++;
+		if (address & (1 << u)) {	// bit 1
+			 ++parity;
 			 AddBitPCF7931(1, tab, l, p);
-		} else{					// bit 0
+		} else {					// bit 0
 			 AddBitPCF7931(0, tab, l, p);
 		}
 	}
-
+	
 	//byte address on 4 bits
-	Dbprintf("Byte address : %02x", byte);
-	for (u=0; u<4; u++)
+	for (u = 0; u < 4; ++u)
 	{
-		if (byte&(1<<u)) {	// bit 1
+		if (byte & (1 << u))
+		{	// bit 1
 			 parity++;
 			 AddBitPCF7931(1, tab, l, p);
-		} else{				// bit 0
-			 AddBitPCF7931(0, tab, l, p);
 		}
+		else				// bit 0
+			 AddBitPCF7931(0, tab, l, p);
 	}
-
+	
 	//data on 8 bits
-	Dbprintf("Data : %02x", data);
 	for (u=0; u<8; u++)
 	{
 		if (data&(1<<u)) {	// bit 1
 			 parity++;
 			 AddBitPCF7931(1, tab, l, p);
-		} else{				//bit 0
+		} 
+		else				//bit 0
 			 AddBitPCF7931(0, tab, l, p);
-		}
 	}
 
-
 	//parity bit
-	if((parity%2)==0){
+	if ((parity % 2) == 0)
 	 	AddBitPCF7931(0, tab, l, p); //even parity
-	}else{
+	else
 		AddBitPCF7931(1, tab, l, p);//odd parity
-	}
 
 	//time access memory
 	AddPatternPCF7931(5120+2680, 0, 0, tab);
 
 	//conversion of the scale time
-	for(u=0;u<500;u++){
-		tab[u]=(tab[u] * 3)/2;
-	}
-
+	for (u = 0; u < 500; ++u)
+		tab[u] = (tab[u] * 3) / 2;
 
 	//compennsation of the counter reload
-	while (!comp){
+	while (!comp)
+	{
```

Comment:
You should stick to one style. Let's have the opening braces on the same line with the `while`